source/patterns/00-atoms/inline/link/link.twig

```diff
@@ -2,7 +2,7 @@
 
 {% spaceless %}
 
-  {%- if attributes|default({}).href is defined -%}
+  {%- if attributes|default({}).href|default(false) -%}
     <a {{- html.attributes(attributes) }}>{{- html.content(text) -}}</a>
   {%- elseif attributes|default({})|length -%}
```

Comment:
This also needs to be changed to ignore `false` values, but can't update to use https://twig.symfony.com/doc/2.x/filters/filter.html due to https://github.com/libero/publisher/issues/190.

Reply:
Just means an empty `<span>` for now.